### PR TITLE
Aggregates Reconcile CLI Implementation

### LINKED ISSUE #767: [FEATURE]: Installer and CLI for `aggregates-reconcile`
### Problem statement

CLI for `aggregates-reconcile`
Linked to #503

### PR DESCRIPTION
**Pull request by vijaypavann-db**
closes #767

### CODE REVIEW

File: labs.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ commands:
       {{end}}
   - name: reconcile
     description: Reconcile is an utility to streamline the reconciliation process between source data and target data residing on Databricks.
+  - name: aggregates-reconcile
+    description: Aggregates Reconcile is an utility to streamline the reconciliation process between the aggregated source and target data residing on Databricks.
```
Comment (sundarshankar89) [inline labs.yml:44]:
```suggestion
    description: Aggregates Reconcile is an utility to streamline the reconciliation process specific metric is compared between source and target data residing on Databricks.
```

File: tests/unit/reconcile/test_runner.py [hunk 2]
```diff
@@ -371,4 +371,78 @@ def test_run_with_failed_execution():
...
+def test_aggregates_reconcile_run_with_job_id_in_state(monkeypatch):
+    monkeypatch.setattr("webbrowser.open", lambda url: None)
+    ws = create_autospec(WorkspaceClient)
+    prompts = MockPrompts(
+        {
+            r"Would you like to open the job run URL .*": "yes",
+        }
+    )
+    installation = MockInstallation(
+        {
```
Comment (sundarshankar89) [inline tests/unit/reconcile/test_runner.py:386]: can you move `state` and `reconcile.yml` as variables

**Review by sundarshankar89 — CHANGES_REQUESTED** [review]
Looks good small nit:

File: tests/unit/reconcile/test_runner.py (hunk 2 above)
Comment (vijaypavann-db) [reply tests/unit/reconcile/test_runner.py:441]: Done

**Review by sundarshankar89 — APPROVED** [review]
LGTM

**Comment by sundarshankar89** [comment]
@bishwajit-db can you take look at this

**Pull request merged**